docs/source/cli/model-training-inference/output-remapping.rst

```diff
@@ -1,6 +1,220 @@
-.. _output-remapping:
+.. _gs-output-remapping:
 
 GraphStorm Output Node ID Remapping
 ====================================
+During :ref:`Graph Construction<graph-construction>`, GraphStorm converts
+user provided node IDs into integer-based node IDs. Thus, the outputs of
+GraphStorm training and inference scripts, i.e., :ref:`saved node
+embeddings<gs-output-embs>` and :ref:`saved prediction results<gs-out-predictions>`,
+are stored with their integer-based node IDs. GraphStorm provides a
+``gconstruct.remap_result`` module to convert the integer-based node IDs
+into the original user provided node IDs according to the :ref:`node ID
+mapping files<gs-id-mapping-files>`.
 
-(Under Construction)
\ No newline at end of file
+.. note::
+
+    If the training or inference tasks are launched by GraphStorm CLIs,
+    the ``gconstruct.remap_result`` module is automatically triggered to
+    to remap the integer-based node IDs into the original user provided
+    node IDs.
+
```

Comment:
Drafted a section for the Input of the remapping command. May add it to give some hints for readers.

```
Remapping inputs
-----------------
Inputs of the node ID remapping are the :ref:`outputs of GraphStorm model training and inference <gs-output>`, which normally include two sets of ``.pt`` files. One set includes files of shuffled integer node IDs, while another includes the corresponding embeddings or predictions in the same row order as the node ID files. 

Below box shows a simple example of node prediction output files.

.. code-block:: bash

    # Saved embedding files and node ID files
    save_embed_path_dir/
        ntype0/
            embed_nids-00000.pt
            embed_nids-00001.pt
            embed-00000.pt
            embed-00001.pt
        ntype1/
            embed_nids-00000.pt
            embed_nids-00001.pt
            embed-00000.pt
            embed-00001.pt

    # Saved prediction files and node ID files
    save_prediction_path_dir/
        ntype0/
            predict_nids-00000.pt
            predict_nids-00001.pt
            predict-00000.pt
            predict-00001.pt
        ntype1/
            predict_nids-00000.pt
            predict_nids-00001.pt
            predict-00000.pt
            predict-00001.pt
    
    # embedding or prediction file contents and node ID fiel contents
    embed_nids-00000.pt | embed-00000.pt                predict_nids-00000.pt | predict.pt
                        |                                                     |
    Graph Node ID       |   embeddings                  Graph Node ID         |   Prediction results
    10                  |   0.112,0.123,-0.011,...      10                    |   0
    1                   |   0.872,0.321,-0.901,...      1                     |   0
    23                  |   0.472,0.432,-0.732,...      23                    |   1

Another set of important input data for remapping is the :ref:`node and edge mapping files <gs-id-mapping-files>`  generated and saved by graph construction pipelines.

```

Reply:
The input formats are introduced in L7 and L8